src/index.js

```diff
@@ -123,7 +123,8 @@ function IpfsAPI (host_or_multiaddr, port) {
   }
 
   self.diag = {
-    net: command('diag/net')
+    net: command('diag/net'),
```

Comment:
Why is it deprecated? :( It has some nice details, is there an alternative to get this data?